src/styl/_components/_preview.styl

```diff
@@ -106,3 +110,158 @@
             left 0
             right 0
             bottom 0
+
+    &-forward // when preview is forward
+        position relative
+
+        .teaser
+            position absolute
+            left 0
+            bottom _rem(30px)
+
+            &-title
+            &-headline
+                color $color-white
+                text-shadow 0 2px 2px $color-base
+
+            &-title
+                position relative
+                z-index 1
+                font-size 1.6em
+                color $color-white
+
+                @media $breakpoints.md
+                    font-size 2.7em
+
+            &-summary
+                display none
+
+        .media
+            .media-wrap
+                &:before
+                    width 4.5em
+                    height 3.5em
+
+                    @media $breakpoints.md
+                        width 6em
+                        height 4.5em
+
+                    left 50%
+                    transform translateX(-50%)
+                    top 0
+                    margin auto
+                    background-size 60% auto
+
+                &:after
+                    content ''
+                    position absolute
+                    left 0
+                    bottom 0
+                    z-index 0
+                    width 100%
+                    height 50%
+                    background linear-gradient(0deg, rgba(0,0,0,.6) 0%, transparent 100%)
```

Comment:
If you talk about the gradient, yes, it is to increase readability.
I apply it to 2:
- A gradient at the bottom of the image to darken this area.
- And a text-shadow on the text to detach it from the background.

Reply:
Ok so what's bothers me a bit is that you applied the gradient on the image and not the container of the text. So, for exemple, if for any reason there is no text on the preview, there will still be the gradient, even if it's not needed.

Reply:
It is not false, but if I apply it on the text container, and the text is not long enough, the block will not be the full width.
![image](https://user-images.githubusercontent.com/19707072/83882232-1d54d380-a742-11ea-87d6-3cb6f715eb2d.png)
This could be resolved by adding a width: 100%;

but, anyway, the text container does not contain the footer bar (with the counterbar).
So if I want a gradient that starts at the bottom of the image, I need to add it over the image.
![image](https://user-images.githubusercontent.com/19707072/83882294-32316700-a742-11ea-8664-331604b1a5fd.png)

Reply:
Oh right I see! Sorry